.github/workflows/ci.yml

```diff
@@ -60,6 +60,13 @@ jobs:
             ${{ matrix.msys-env }}-clang-tools-extra
             ${{ (matrix.config && format('{0}-qt6-base', matrix.msys-env)) || '' }}
 
+      - name: Install Qt
+        if: ${{ !!matrix.msvc && matrix.config }}
+        uses: jurplel/install-qt-action@v3
+        with:
+          version: '6.4.2'
```

Comment:
Why using an explicit version?
It currently defaults to `6.8.3` when this key is not present.

Reply:
```suggestion
          version: '6.*'
```

Reply:
Yeah, trying that right now :)